client/apps/city_issues/templates/user/user.html

```diff
@@ -0,0 +1,124 @@
+                        </div>
+                        <div class=" col-md-9 col-lg-9 ">
+                            <table class="table table-user-information">
+                                <tbody>
+                                <tr>
+                                    <td>Alias:</td>
+                                    <td>{{ user.alias }}</td>
+                                </tr>
+                                <tr>
+                                    <td>Role:</td>
+                                    <td>{{ user.role.role }}</td>
+                                </tr>
+                                <tr>
+                                    <td>Email</td>
+                                    <td><a href="mailto:info@support.com">{{ user.email }}</a>
+                                    </td>
+                                </tr>
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                </div>
+                <div class="panel-footer">
+                    <span class="pull-right">
+                        <a href="#edit.html" class="btn btn-sm btn-warning">
+                            <i class="glyphicon glyphicon-edit"></i>
+                        </a>
+                    <a href="/" class="btn btn-sm btn-danger">
+                        <i class="glyphicon glyphicon-remove"></i>
+                    </a>
+                    </span>
+                </div>
+            </div>
+        </div>
+        <div class="width">
+            <h2>Added issus</h2>
+
+            <table id="user_issues" class="display" width="100%" cellspacing="0">
+                <thead>
+                <tr>
+                    <th>#</th>
+                    <th>Title</th>
+                    <th>Category</th>
+                    <th>Status</th>
+                    <th>Open date</th>
+                    <th>Close date</th>
+                    <th></th>
+                </tr>
+                </thead>
+                <tfoot>
+                <tr>
+                    <th>#</th>
+                    <th>Title</th>
+                    <th>Category</th>
+                    <th>Status</th>
+                    <th>Open date</th>
+                    <th>Close date</th>
+                    <th></th>
+                </tr>
+                </tfoot>
+                <tbody>
+                {% for isssue in user_issues %}
+                    <tr>
+                        <td>{{ forloop.counter }} </td>
+                        <td>{{ isssue.issue.title }}</td>
+                        <td>{{ isssue.issue.category.category }}</td>
+                        <td>{{ isssue.issue.status }}</td>
+                        <td>{{ isssue.issue.open_date }}</td>
+                        <td>{{ isssue.issue.close_date }}</td>
```

Comment:
If you change query then here will be so
`{% for issue in user_issues %}
    <tr>   <td>{{ forloop.counter }} </td>
    <td>{{ issue.title }}</td> ... `